### PR TITLE
Add questionnaire function to init options

### PR DESCRIPTION
**Pull request by nporteschaikin**

### CODE REVIEW

File: lib/api/new.coffee [hunk 1]
```diff
@@ -46,6 +46,12 @@ class New
 
     opts =
       locals: opts.overrides ? {}
+      questionnaire: (questions, skip) ->
+        W.promise (resolve, reject) ->
+          qs = []
+          for question in questions
+            qs.push(question) unless !_.contains(skip, question.name)
```
Comment (kylemac) [inline lib/api/new.coffee]: I think you mean `qs.push(question) unless _.contains(skip, question.name)`  (gets rid of the double negative)

**Comment by kylemac** [comment]
@nporteschaikin 1 inline comment and you just need to do a `inquirer = require 'inquirer` in `lib/api/new.coffee` and this is working

File: lib/api/new.coffee (hunk 1 above)
Comment (nporteschaikin) [inline lib/api/new.coffee]: @kylemac correct.  fixing + rebasing now.

**Comment by nporteschaikin** [comment]
@kylemac all set.

**Comment by kylemac** [comment]
:+1: merging this in and going to continue to put `origin/sprout-upgrade` through it's paces

**Pull request merged**